fast64_internal/f3d/f3d_material.py

```diff
@@ -1091,6 +1097,15 @@ def ui_procAnimVecEnum(procAnimVec, layout, name, vecType, useDropdown):
 			if combinedOption == "Rotation":
 				pass
 	
+	if (useTex0 or useTex1):
+		layout.box().label(text = 'SM64 SetTileSize Texture Scroll')
+
+		if (useTex0):
+			ui_tileScroll(material.tex0, 'Texture 0 Speed', layout)
+
+		if (useTex1):
```

Comment:
```suggestion
	if useTex0 or useTex1:
		layout.box().label(text = 'SM64 SetTileSize Texture Scroll')

		if useTex0:
			ui_tileScroll(material.tex0, 'Texture 0 Speed', layout)

		if useTex1:
```
again, not familiar with fast64, what makes it so that this doesn't get drawn in "oot mode"?

Reply:
lol looks like programming in C and js all day has leaked into my code, where are my semicolons?